apigee/src/main/apigee/apiproxies/data/apiproxy/targets/default.xml

```diff
@@ -2,6 +2,6 @@
 <TargetEndpoint name="default">
   <HTTPTargetConnection>
     <Properties/>
-    <URL>http://dashboard.climateiq.org/internal/data/</URL>
+    <URL>https://dashboard.climateiq.org/data/</URL>
```

Comment:
hm cant anyone hit this endpoint? the apigee targets should only be accessible to the Apigee instance